CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+- All redundancy finder methods will return four items --- redundant groups, bin centers, lengths, and conjugates.
+- UVData.get_antenna_redundancies will no longer automatically conjugate baselines.
+- UVData.get_baseline_redundancies and UVData.get_antenna_redundancies have been combined.
```

Comment:
The wording in these two lines is weird -- you're calling out a change to a method that is being deprecated and the new method isn't mentioned.

Reply:
I've moved this to the Added section and rephrased it to say that the new function replaces the old two.